packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,108 @@
+### Code Considerations: What do engineers need to do?
+
+<div aria-label="Example Demo">
+    <DividerExampleAccess />
+</div>
+
+> ```jsx
+> const date1 = '1979 Cerner is founded';
+> const date2 = '2006 Cerner opens Healthe Clinic';
+>
+> const DividerExampleAccess = () => (
+> <div>
+>    <Card>
+>        <Card.Body>
+>            <div>
+>                <p>
+>                    For over four decades, Cerner has pursued breakthrough innovation that’s helped shape the health care
+>                    industry as we know it.
+>                </p>
+>                <Divider text={date1} level={3} />
+>                <p>
+>                    Before leaving Arthur Andersen & Co. to form their own company, three young friends and co-workers
+>                    sat around a picnic table at Loose Creek Park in Kansas City, Missouri brainstorming business
+>                    ideas. It was here that Neal Patterson, Paul Gorup and Cliff Illig conceived PGI & Associates, the
+>                    company that would eventually become Cerner.
+>                </p>
+>                <Divider text={date2} level={3} />
+>                <p>The company’s first on-site health clinic opened on its World Headquarters campus.</p>
+>                <Divider />
+>                <p>
+>                    Since 1979, we’ve been proud to work at the intersection of health care and information technology to
+>                    connect people and systems around the world.
+>                </p>
+>            </div>
+>        </Card.Body>
+>    </Card>
+> </div>
+> );
+> ```
+
+-  Engineers must work with content creators to ensure heading levels for all Dividers are appropriate for their placement on the page.
+  - Dividers with text cannot be heading level 1. Heading level 1 is reserved for the page’s title.
+  - Engineers must assign the proper heading level prop (H2-H6) to a Divider.
+  - Dividers with text must follow the correct heading level order based on its relationship to other headings in the flow of the page’s content.
+
+### Content Considerations: What do content creators need to do?
+
+- Dividers with text must act as a heading to describe the section.
+  - Ensure that the divider text accurately describes the content it represents.
+- Content creators must provide engineers with the desired heading text.
+  - Work with engineers to understand the content structure of the page to ensure heading levels for all Dividers are appropriate for their placement within the page.
+      - Dividers with text should never be a heading level 1.
+- Best practice dictates that headings should be used to identify sections of content.
+  - If a Divider is used with text, the Divider text will act as the appropriate heading.
+  - If no heading text is provided within the Divider, best practice would be to include a heading within the content after the divider.
+      - Dividers with no text should not have a heading level assigned to the Divider itself.
+
+
+
```

Comment:
nit: Extra lines

Reply:
removed in https://github.com/cerner/terra-core/pull/3677/commits/90dbc52654717fe1c86bc3f5c6788289f093c9b3